docs/packages.rst

```diff
@@ -0,0 +1,92 @@
+Via Package Manager
+===================
+
+You can install xonsh using ``conda``, ``pip`` or the package manager for 
+your operating system distribution.
+
+For the fullest interactive user experience, these additional packages should also be installed:
+
+  :prompt-toolkit: for command completion, configurable key bindings and especially multi-line line editing.
+  :pygments: for xonsh and python syntax-specific highlighting
+  :setproctitle: updates process title (in terminal window and process monitor) to match Xonsh arguments.
+
+Installing with these packages is the recommended configuration and is documented first.   
+If you are operating in a specialized or restricted environment, you can install just the xonsh package, as 
+described in `fewer prerequisites`_
+
+
+**conda:**
+
+.. code-block:: console
+
+    $ conda config --add channels conda-forge
+    $ conda install pygments prompt-toolkit setproctitle xonsh
```

Comment:
I think the conda-forge package includes those as deps, so this just needs to be `conda install xonsh`

Reply:
OK, I'll doc it.
But how do you conda install without the extra packages, e.g for emacs?
If you can't (which is what it looks like), why can't we just do it the same way for vanilla pip?